composer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "salcode/bootstrap-genesis",
+    "description": "WordPress Genesis Child Theme setup to use Bootstrap, Sass, and Grunt",
+    "keywords": ["wordpress", "theme"],
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Sal Ferrarello",
+            "email": "sal@ironcodestudio.com",
+            "homepage": "http://salferrarello.com/"
+        }
+    ],
+    "autoload": {
```

Comment:
## "type": "wordpress-theme"

Good catch, yes we'll need 

```
"type": "wordpress-theme",
```

I missed that on my first pass.

## autoload value

Just verified, we do __not__ need the `autoload` property at all in the `composer.json`.